packages/e2e-test-utils/README.md

```diff
@@ -399,6 +424,26 @@ _Returns_
 
 -   `Promise<?string>`: Promise resolving to a string or null, depending whether a page error is present.
 
+### getSiteEditorMenuItem
+
+Searches for an item in the site editor navigation menu with the provided label.
+
+_Parameters_
+
+-   _label_ `string`: The label to search the menu item for.
+
+_Returns_
+
+-   `Promise<?ElementHandle>`: The menu item handle or `null`
+
+### goToSiteEditor
```

Comment:
**suggestions (non-blocking):** Maybe we follow the general `visit*` pattern for similar actions.